### PR TITLE
Loki Context UI: Do not disable last label

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

This PR improves two things in Loki's experimental context UI:
- description
- can not remove labels if it would be the last enabled one

**Steps to reproduce**

1. Configure Grafana to use the `logsContextDatasourceUi` feature flag
2. Start Grafana and a Loki DB
3. Make any query containing a line filter operation
4. Open the LogContext and try to remove labels

Before:

https://user-images.githubusercontent.com/8092184/217248496-9b78d962-e13f-4233-a531-c5fab72040e9.mov

With the fix:

https://user-images.githubusercontent.com/8092184/217248546-62304e3f-a2da-4aa9-a95c-d7dab62acfd3.mov

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { css } from '@emotion/css';
+import { cloneDeep } from 'lodash';
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiContextUi.tsx:2]: `structuredClone`?

Comment (matyax) [reply public/app/plugins/datasource/loki/components/LokiContextUi.tsx:2]: I wouldn't be surprised if at some point a "remove lodash" task is created for Grafana as a whole.

**Comment by matyax** [comment]
FYI missing steps to reproduce.

**Review by matyax — APPROVED** [review]
I will have to trust you on this.

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx (hunk 1 above)
Comment (svennergr) [reply public/app/plugins/datasource/loki/components/LokiContextUi.tsx]: Oh, TIL! Thanks so much!

**Comment by svennergr** [comment]
> FYI missing steps to reproduce.

In case you want to repro, I added some steps above.

**Review by matyax — APPROVED** [review]
Much clear now! Thank you!

**Review by matyax — APPROVED** [review]

**Pull request merged**